knimin/lib/qiita_jira_util.py

```diff
@@ -345,20 +347,54 @@ def create_sequencing_run(pool_id, email, sequencer, reagent_type,
         The pool being sequenced
     email : str
         The email of the user preparing the run
-    sequencer : id
-        The sequencer id
     reagent_type : str
         The reagent type
     reagent_lot : str
         The reagent lot
+    platform : str
+        The sequencing platform (e.g., Illumina)
+    instrument_model : str
+        The model of the instrument (e.g., MiSeq)
+    assay : str
+        The assay used (e.g., Kapa Hyper Plus)
+    fwd_cycles : int
+        The number of forward cycles used.
+    rev_cycles : int
+        The number of reverse cycles used.
 
     Returns
     -------
     int
         The run id
     """
-    run_id = db.create_sequencing_run(pool_id, email, sequencer, reagent_type,
-                                      reagent_lot)
+    # Store the information on the DB
+    # Sequencer is hardcoded to None. This information may bot be known at this
+    # time but we can potentially retrieve this from the output sequencing
+    # folder.
+    run_id = db.create_sequencing_run(
+        pool_id, email, None, reagent_type, reagent_lot, platform,
+        instrument_model, assay, fwd_cycles, rev_cycles)
+
+    run = db.read_sequencing_run(run_id)
+    pool = db.read_pool(run['pool_id'])
+
+    # Write the sample sheet
+    instrument_type = 'miseq' if instrument_model == 'MiSeq' else 'hiseq'
```

Comment:
Okay. This and the below possibly makes it more difficult to add in another instrument, like the novaseq. But I think that's fine for the time being...